<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings3DDockWidget</class>
 <widget class="QDockWidget" name="Settings3DDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3D settings</string>
  </property>
  <widget class="QWidget" name="dockWidgetContentsSettings3D">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>3D interpolation:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBoxInterpolationMode">
      <item>
       <property name="text">
        <string>GL_LINEAR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GL_NEAREST</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBoxVolumeRendering">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Volume rendering</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="2" colspan="2">
        <widget class="QSlider" name="horizontalSliderVRSlices">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>1200</number>
         </property>
         <property name="value">
          <number>500</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QComboBox" name="comboBoxVolumeRenderingMode">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Alpha blending</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Maximum intesity projection</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Minimum intensity projection</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Average intensity projection</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Resolution:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="spinBoxVRSlices">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>3</number>
         </property>
         <property name="maximum">
          <number>1200</number>
         </property>
         <property name="value">
          <number>500</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Mode:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBoxVRSlices</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderVRSlices</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderVRSlices</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxVRSlices</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
